codeserver/ubi9-python-3.12/httpd/httpd.conf

```diff
@@ -0,0 +1,41 @@
+# Basic httpd configuration for CGI processing
+ServerRoot "/etc/httpd"
+Listen 8080
+
+# Load modules
+LoadModule mpm_prefork_module modules/mod_mpm_prefork.so
+LoadModule authz_core_module modules/mod_authz_core.so
+LoadModule dir_module modules/mod_dir.so
+LoadModule cgi_module modules/mod_cgi.so
+LoadModule rewrite_module modules/mod_rewrite.so
+
+# User and Group - run as the container user
+User 1001
+Group 0
```

Comment:
this has no effect at best, no? openshift assigns random uid

Reply:
Right, removed